### ISSUE TITLE
[not a bug] Lovelace display tips 

### ISSUE THREAD

**Issue by tbrasser** [gieljnssns/buienalarm-sensor-homeassistant#1]
I was wondering how people are showing this component's data in Lovelace.

Currently I have just put the temperature sensor in a history graph, but I'd love to also have a precipitation overview. 

ideas?

**Comment by gieljnssns** [comment]
I think this is the way to go for you. 
https://community.home-assistant.io/t/buienalarm-visuals-in-a-panel-iframe/120558

**Comment by tbrasser** [comment]
Yeah I saw that, but that doesn't make use of your component right? Maybe I can use mini graph card to display the precipitation for the next x hours.

**Comment by gieljnssns** [comment]
If you find something like that, you can share your code…
I just use it to close my awnings when there is rain in the next x.
